ApexCare/ApexCare/BusinessLogic/Communication/EmailCommunication.cs

```diff
@@ -0,0 +1,17 @@
+using ApexCare.BusinessLogic.Communication.Interfaces;
+
+namespace ApexCare.BusinessLogic.Communication{
+    public class EmailCommunication : ICommunication{
+        private string[] Message;
+        public EmailCommunication(){
+            Message = new string[] {};
+        }
+        public void promptInfo(){
+            System.Console.WriteLine("Prompting the user...");
```

Comment:
??????? Why console.writeline when communicating with user ?? Polluting the logs